ec2/spark_ec2.py

```diff
@@ -349,6 +351,16 @@ def launch_cluster(conn, opts, cluster_name):
     if opts.identity_file is None:
         print >> stderr, "ERROR: Must provide an identity file (-i) for ssh connections."
         sys.exit(1)
+
+    if not os.path.exists(opts.identity_file):
+        print >> stderr, "ERROR: The identity file '{f}' doesn't exist.".format(f=opts.identity_file)
+        sys.exit(1)
+
+    file_mode = os.stat(opts.identity_file).st_mode
+    if not (file_mode & os.stat.S_IRUSR):
+        print >> stderr, "ERROR: Must set the permissions for the identity file to S_IRUSR."
```

Comment:
End users won't understand what `S_IRUSR` means. It's better to say something more straightforward like, "The identity file must be readable only by the owner."